src/views/lists_home.js

```diff
@@ -151,32 +160,68 @@ class Icons extends Component {
           {list2.map((l, i) => (
             <ListItem
               roundAvatar
-              avatar={{ uri: l.avatar_url }}
+              leftAvatar={{ source: { uri: l.avatar_url } }}
               key={i}
               onPress={log}
               title={l.name}
               subtitle={l.subtitle}
+              disclosure
+              bottomDivider
             />
           ))}
         </List>
         <List>
-          {
-            list2.map((l, i) => (
-              <ListItem
-                key={i}
-                leftIcon={{ name: 'user-circle-o', type: 'font-awesome', style: {color: 'blue'} }}
-                title={l.name}
-                titleStyle={{color: 'red'}}
-                subtitle={l.subtitle}
-                rightTitle='11:00am'
-                rightTitleStyle={{color: 'green'}}
-              />
-            ))
-          }
+          {list2.map((l, i) => (
+            <ListItem
+              key={i}
+              leftIcon={{
+                size: 40,
+                name: 'user-circle-o',
+                type: 'font-awesome',
+                color: 'blue',
+              }}
+              title={l.name}
+              titleProps={{ children: l.name, style: { color: 'red' } }}
+              subtitle={l.subtitle}
+              rightTitle="11:00am"
+              rightTitleProps={{ style: { color: 'green' } }}
+              bottomDivider
+            />
+          ))}
+        </List>
+        <List>
+          <ListItem
+            title="Name"
+            inputProps={{ placeholder: 'Type your name' }}
+            disclosure
+            bottomDivider
+          />
+          <ListItem
+            title="Switch that please 😲"
+            switchProps={{}}
+            bottomDivider
+          />
```

Comment:
```
<ListItem
  title="Switch that please 😲"
  switchProps={{}}
  bottomDivider
/>
```

I see the pattern here where just passing `switchProps` or `buttonGroupProps` props is enought. I personally think we should do `switch` or `buttonGroup` props that are needed for those component and then have `Props` be the optional

thoughts?

Reply:
See the comment above

Reply:
cool. Let's make it 1 `switch` prop and pass in everything within it.. like you have in the above comment.